### ISSUE TITLE
Youtube ads no longer being blocked on videos

### ISSUE THREAD

**Issue by lolwulf2** [dhowe/AdNauseam#2544]
### This is an AdNauseam-specific issue.

- [X] I have tested the same page on *uBlock Origin*, and the Ad is not visible.

### I have tried to reproduce the issue when...

- [X] AdNauseam is the only extension
- [X] AdNauseam uses default lists and setting.
- [X] Using a new, unmodified browser profile
- [X] All filter lists are updated

### Does Strict-Block mode solves the issue?

- [X] I have tested the same page with Strict-Block enabled, and the Ad is still visible.

### URL(s) where the issue occurs.

https://www.youtube.com/

### Your location/country.

Denmark

### Screenshot(s)

![image](https://github.com/user-attachments/assets/6a490298-1c47-4f7b-87de-628e6734b80b)

**Comment by lolwulf2** [comment]
it is the same with ublock origin

**Comment by Codrineye** [comment]
Ah, this is nice, I was just looking in the documentation in search of any exception for youtube ad blocking.
Good to know it's not just on my end

**Comment by metapea** [comment]
same here

**Comment by mrocha98** [comment]
same here, using Firefox Developer Edition

**Comment by mneunomne** [comment]
The release fixing this should be rolling out soon, sorry for the inconvenience.

**Comment by lolwulf2** [comment]
cheers 👍  so far it seems to be working again so does ublock will monitor for today as well

**Comment by Codrineye** [comment]
Well, that's a problem, I still get youtube ads

**Comment by mneunomne** [comment]
Latest release should be fixing all youtube related issue: https://github.com/dhowe/AdNauseam/releases/tag/v3.22.0

**Comment by noidedxyz** [comment]
Every video I click on now I see 2 ads back to back. Not much variety, either just 1 of 3 of the same ads. I'm on Brave.

**Comment by Codrineye** [comment]
hey hey, this is rather annoying
![image](https://github.com/user-attachments/assets/daa780ae-8493-4219-918f-066d4f458407)
and I have the most up to date version
![image](https://github.com/user-attachments/assets/e8a9f9ac-2e50-4660-a321-689ef26c12e0)

**Comment by dhowe** [comment]
> hey hey, this is rather annoying

do you want to submit a PR ?

**Comment by Codrineye** [comment]
nope, just bringing attention to it

**Comment by mneunomne** [comment]
@noidedxyz @Codrineye are you still experiencing this issue? It happens sometimes that youtube update their scripts and the filter lists take a bit of time to catch-up, this happens sometimes also with uBlock Origin.

**Comment by Codrineye** [comment]
still happen, but now it seems to be random
I guess I changed something in the filter

**Comment by mneunomne** [comment]
I couldn't reproduce it, but I'll keep testing. Thank you for update.

**Comment by noidedxyz** [comment]
> @noidedxyz @Codrineye are you still experiencing this issue? It happens sometimes that youtube update their scripts and the filter lists take a bit of time to catch-up, this happens sometimes also with uBlock Origin.

I am no longer having this issue. It seemingly was only for a couple of days that this was happening to me.

**Comment by BamsTheSergal** [comment]
i've been recently getting really slow preformance and sometimes full video stoppages, only after watching a ad would it work again.

**Comment by bob-carter** [comment]
Not sure if it's the same issue, but I have done what the OP has done, and when I have the program running on my firefox Youtube videos will freeze, or skip around. I believe it is because of the new embedding of ads into the video encoding or stream, that I heard about.

**Comment by CookieSalesman** [comment]
Don't really want this to be a full issue submission but still wanted to mention it somewhere. FYI I have tons of extensions and disabled nothing. One of them is DeArrow. It was failing to work on youtube. After turning disabling adnauseum (temporarily) on YT, dearrow worked. The 'open submission' button worked after this.

![image](https://github.com/user-attachments/assets/21377013-63c9-41d6-a9b6-23175f004b79)

**Comment by FishRSA** [comment]
I was on 3.22 for a while and today I started seeing ads(in the player before a video starts, nothing else) on youtube, like non stop every single time I refresh the page I get 2 ads back to back. I updated to 3.23.2 and it made no difference. Anyone else? Anything I can do to fix it?

**Comment by VandilFish** [comment]
Have been getting ads on YouTube for a couple weeks now on 3.23.2. have tried everything. reinstalling extension and even browser. I have to refresh 4-8 times before the ads gets blocked.

**Comment by mneunomne** [comment]
Working now on Merge to uBlock 1.61.0 in order to fix this issue: https://github.com/dhowe/AdNauseam/pull/2575

**Comment by metapea** [comment]
I've been getting ads again

**Comment by bphd** [comment]
Same

**Comment by dhowe** [comment]
status on this @mneunomne ? thanks

**Comment by bphd** [comment]
Stills happens on 24 afaik, but let's see such status

**Comment by mneunomne** [comment]
Making new merge now with uBlock 1.62.0 that should fix this. Firefox store hasnt updated our previous version yet, seems there are some end of year holidays in their approval process.

**Comment by theSectClown** [comment]
+

![Image](https://github.com/user-attachments/assets/2a5c1413-2651-4e85-a401-7016d5a36f52)

![Image](https://github.com/user-attachments/assets/33dc2498-cf0e-4070-972b-36cbcffbe4b8)

![Image](https://github.com/user-attachments/assets/2b74e5e9-df82-4a37-8ddb-fd253d11af17)

**Comment by Ciprian-eL** [comment]
AdNauseamv3.25.4 - Firefox 140.0.1
The youtube popup (ad-blocker detected) appeares for a fraction, disappears, then video is unplayable. The youtube play button changes into the pause symbol when clicked, and back, if clicked again, but nothing plays.

![Image](https://github.com/user-attachments/assets/2c8abcc9-9481-4fc8-8ab8-5f3fb51872ab)

**Comment by bob-carter** [comment]
> AdNauseamv3.25.4 - Firefox 140.0.1 The youtube popup (ad-blocker detected) appeares for a fraction, disappears, then video is unplayable. The youtube play button changes into the pause symbol when clicked, and back, if clicked again, but nothing plays.
> 
> ![Image](https://github.com/user-attachments/assets/2c8abcc9-9481-4fc8-8ab8-5f3fb51872ab)

Unsure this is what's happening, but I also have noticed it, this only happens for Age Restricted Videos for me, and if I turn ABS (Adnasumium) off it still refuses to work.

**Comment by MatthewMBetley** [comment]
AdNauseam 3.27.0 - Firefox 147.0.1

Ads before and _during_ YouTube videos started leaking through again today, even with all my settings staying exactly the same. Strict mode didn't help either. There's a _very_ slight chance it might be interference with the Unhook extension that I also run, but I didn't make any changes there either so it's almost like YouTube updated mid-video to force ads through. Very tempted to touch more grass just to spite Google.

<img width="1898" height="837" alt="Image" src="https://github.com/user-attachments/assets/62e3bd47-46de-4bb3-a406-eecdd7748261" />

**Comment by Chronogram** [comment]
<img width="1264" height="710" alt="Image" src="https://github.com/user-attachments/assets/09277865-cdcc-4ed8-9dfc-11eebcd39c53" />

**Comment by digital-idiot** [comment]
<img width="1028" height="909" alt="Image" src="https://github.com/user-attachments/assets/46201d32-3575-4530-9c1f-0ae07e1459eb" />

All videos stop playing after approx 30s.


* Browser: Firefox 148.0.2 (64-bit)
* OS: macOS Tahoe (26.3.1)

**Comment by blueheron786** [comment]
Been using Adnauseum for about a week. I'm in SlimJet (Chromium based), with no other extensions (other than a password manager). Tried with strict mode on and off; it made no difference. Private mode, same thing.

It was fine all week. And today, I get sponsored video ads again...

Tried Incognito, same result.

<img width="1577" height="759" alt="Image" src="https://github.com/user-attachments/assets/987aad97-3b9c-4c33-8a16-76463d993594" />

**Comment by dhowe** [comment]
ADN Version? OS/version?

**Comment by blueheron786** [comment]
Interesting, it's no longer reproducible on that system. I didn't change anything.

- ADN v3.28.6
- OS: Windows 11 Enterprise version 25H2
- Browser: SlimJet Version 47.0.0.0 (based on Chromium 136.0.7103.114) (Official Build) (64-bit)

**Comment by mneunomne** [comment]
This is normal since the filter lists are constantly being updated.

**Comment by blueheron786** [comment]
New issue today, I'm getting ads mid-video. This is a first for me.
